drivers/clock_control/clock_control_nrf.c

```diff
@@ -84,6 +85,10 @@ static uint64_t hf_stop_tstamp;
 static struct onoff_client lfsynth_cli;
 #endif
 
+#if CONFIG_CLOCK_CONTROL_NRF_HFINT_CALIBRATION_PERIOD
+static struct onoff_client hf_cal_cli;
```

Comment:
Why not have this together with `calibration_timer` definition below?

Reply:
rearranged location of variables/functions